.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea/
```

Comment:
This should be placed in your global gitignore :)

Reply:
Done :-)